backend/app/assets/javascripts/spree/backend/shipments.js

```diff
@@ -193,7 +113,7 @@ addVariantFromStockLocation = function(stock_location_id, variant_id, quantity)
       url: Spree.routes.shipments_api,
       data: {
         shipment: {
-          order_id: order_number
+          order_id: window.order_number
```

Comment:
Yes and to ensure that always the global value is used and never a local one. Globals are bad, but it was global befor this change, so it's ok to make it more explicit here. We should change that on future versions to `Spree.current_order_number`